components/automate-chef-io/Makefile

```
@@ -11,6 +11,15 @@ clean:
 sync: themes/chef
 	pushd themes/chef && git fetch && git reset --hard origin/master && popd
 
+SWAGGER_DIR=data/docs/api_chef_automate
+
+clean_swagger_files:
+	rm -rf $(SWAGGER_DIR)/*
```

This looks scary -- can we play it safe with
```suggestion
	rm -rf ./$(SWAGGER_DIR)/*
```